bindings/python/capstone/arc_const.py

```diff
@@ -0,0 +1,297 @@
+ARC_INS_ABS_S = 58
+ARC_INS_ADD1_S = 59
+ARC_INS_ADD2_S = 60
+ARC_INS_ADD3_S = 61
+ARC_INS_AND_S = 62
+ARC_INS_BIC_S = 63
+ARC_INS_BRK_S = 64
+ARC_INS_EXTB_S = 65
+ARC_INS_EXTH_S = 66
+ARC_INS_JEQ_S = 67
+ARC_INS_JL_S = 68
+ARC_INS_JL_S_D = 69
+ARC_INS_JNE_S = 70
+ARC_INS_J_S = 71
+ARC_INS_J_S_D = 72
+ARC_INS_LSR_S = 73
+ARC_INS_MPYUW_S = 74
+ARC_INS_MPYW_S = 75
+ARC_INS_MPY_S = 76
+ARC_INS_NEG_S = 77
+ARC_INS_NOP_S = 78
+ARC_INS_NOT_S = 79
+ARC_INS_OR_S = 80
+ARC_INS_SEXB_S = 81
+ARC_INS_SEXH_S = 82
+ARC_INS_SUB_S = 83
+ARC_INS_SUB_S_NE = 84
+ARC_INS_SWI_S = 85
+ARC_INS_TRAP_S = 86
+ARC_INS_TST_S = 87
+ARC_INS_UNIMP_S = 88
+ARC_INS_XOR_S = 89
+ARC_INS_LDB_S = 90
+ARC_INS_LDH_S = 91
+ARC_INS_J = 92
+ARC_INS_JL = 93
+ARC_INS_JLI_S = 94
+ARC_INS_LDB_AB = 95
+ARC_INS_LDB_AW = 96
+ARC_INS_LDB_DI_AB = 97
+ARC_INS_LDB_DI_AW = 98
+ARC_INS_LDB_DI = 99
+ARC_INS_LDB_X_AB = 100
+ARC_INS_LDB_X_AW = 101
+ARC_INS_LDB_X_DI_AB = 102
+ARC_INS_LDB_X_DI_AW = 103
+ARC_INS_LDB_X_DI = 104
+ARC_INS_LDB_X = 105
+ARC_INS_LDB = 106
+ARC_INS_LDH_AB = 107
+ARC_INS_LDH_AW = 108
+ARC_INS_LDH_DI_AB = 109
+ARC_INS_LDH_DI_AW = 110
+ARC_INS_LDH_DI = 111
+ARC_INS_LDH_S_X = 112
+ARC_INS_LDH_X_AB = 113
+ARC_INS_LDH_X_AW = 114
+ARC_INS_LDH_X_DI_AB = 115
+ARC_INS_LDH_X_DI_AW = 116
+ARC_INS_LDH_X_DI = 117
+ARC_INS_LDH_X = 118
+ARC_INS_LDH = 119
+ARC_INS_LDI_S = 120
+ARC_INS_LD_AB = 121
+ARC_INS_LD_AW = 122
+ARC_INS_LD_DI_AB = 123
+ARC_INS_LD_DI_AW = 124
+ARC_INS_LD_DI = 125
+ARC_INS_LD_S_AS = 126
+ARC_INS_LD = 127
+ARC_INS_LEAVE_S = 128
+ARC_INS_LR = 129
+ARC_INS_LSR_ = 130
+ARC_INS_LSR_F = 131
+ARC_INS_LSR = 132
+ARC_INS_MAX_ = 133
+ARC_INS_MAX_F = 134
+ARC_INS_MAX = 135
+ARC_INS_MIN_ = 136
+ARC_INS_MIN_F = 137
+ARC_INS_MIN = 138
+ARC_INS_MOV_S_NE = 139
+ARC_INS_MOV_ = 140
+ARC_INS_MOV_F = 141
+ARC_INS_MOV = 142
+ARC_INS_MPYMU_ = 143
+ARC_INS_MPYMU_F = 144
+ARC_INS_MPYMU = 145
+ARC_INS_MPYM_ = 146
+ARC_INS_MPYM_F = 147
+ARC_INS_MPYM = 148
+ARC_INS_MPY_ = 149
+ARC_INS_MPY_F = 150
+ARC_INS_MPY = 151
+ARC_INS_NORMH_F = 152
+ARC_INS_NORMH = 153
+ARC_INS_NORM_F = 154
+ARC_INS_NORM = 155
+ARC_INS_OR_ = 156
+ARC_INS_OR_F = 157
+ARC_INS_OR = 158
+ARC_INS_POP_S = 159
+ARC_INS_PUSH_S = 160
+ARC_INS_ROR_ = 161
+ARC_INS_ROR_F = 162
+ARC_INS_ROR = 163
+ARC_INS_RSUB_ = 164
+ARC_INS_RSUB_F = 165
+ARC_INS_RSUB = 166
+ARC_INS_SBC_ = 167
+ARC_INS_SBC_F = 168
+ARC_INS_SBC = 169
+ARC_INS_SETEQ_ = 170
+ARC_INS_SETEQ_F = 171
+ARC_INS_SETEQ = 172
+ARC_INS_SEXB_F = 173
+ARC_INS_SEXB = 174
+ARC_INS_SEXH_F = 175
+ARC_INS_SEXH = 176
+ARC_INS_STB_S = 177
+ARC_INS_ST_S = 178
+ARC_INS_STB_AB = 179
+ARC_INS_STB_AW = 180
+ARC_INS_STB_DI_AB = 181
+ARC_INS_STB_DI_AW = 182
+ARC_INS_STB_DI = 183
+ARC_INS_STB = 184
+ARC_INS_STH_AB = 185
+ARC_INS_STH_AW = 186
+ARC_INS_STH_DI_AB = 187
+ARC_INS_STH_DI_AW = 188
+ARC_INS_STH_DI = 189
+ARC_INS_STH_S = 190
+ARC_INS_STH = 191
+ARC_INS_ST_AB = 192
+ARC_INS_ST_AW = 193
+ARC_INS_ST_DI_AB = 194
+ARC_INS_ST_DI_AW = 195
+ARC_INS_ST_DI = 196
+ARC_INS_ST = 197
+ARC_INS_SUB1_ = 198
```

Comment:
```suggestion
ARC_INS_SUB1_ = 198
```

These instruction ids wnding in `_` look fishy. I suspect they end in a `.` or with a `{`. But they for sure end with a non alphabetical character. Can you check if this is expected? For some archs a `.` actually belongs to the mnemonic (PPC). For some it doesn't (ARM).

If the mnemonic should not end in a special character, you can check the `getNormalizedMnemonic` function in the `PrinterCapstone`. You can remoce special chars there.

Reply:
It works like that because of instructions like `sub1.$cc`. I can propose to replace $cc for ""

Reply:
Yes, please remove it.